deploy/evmos/130_check_TBTCMetaPoolTokens.ts

```diff
@@ -0,0 +1,38 @@
+import { DeployFunction } from "hardhat-deploy/types"
+import { HardhatRuntimeEnvironment } from "hardhat/types"
+import { isTestNetwork } from "../../utils/network"
+import { BigNumber } from "ethers"
+
+const USD_TOKENS_ARGS: { [token: string]: any[] } = {
+  TBTCv2: ["tBTC v2", "tbtc", "18"],
```

Comment:
symbol is `tBTC` as per https://etherscan.io/address/0x18084fba666a33d37592fa2633fd49a74dd93a88#readContract